test/e2e/apps/statefulset.go

```diff
@@ -45,6 +42,9 @@ import (
 	e2eservice "k8s.io/kubernetes/test/e2e/framework/service"
 	e2estatefulset "k8s.io/kubernetes/test/e2e/framework/statefulset"
 	imageutils "k8s.io/kubernetes/test/utils/image"
+	"strings"
+	"sync"
+	"time"
```

Comment:
can you keep the separation, go fmt should do it

Reply:
Changes as requested. Thank you for the review.